infra/lib/fhir-server-stack.ts

```diff
@@ -74,8 +74,8 @@ export function settings(): Settings {
   }
   return {
     ...defaults,
-    cpu: 0.5 * vCPU,
-    memoryLimitMiB: 1024,
+    cpu: 1 * vCPU,
+    memoryLimitMiB: 2048,
```

Comment:
0.5 vCPU was not being enough to start the service - health checks were failing, even adjusting those thresholds - [context](https://metriport.slack.com/archives/C04DMKE9DME/p1722359008629989?thread_ts=1722348020.483229&cid=C04DMKE9DME)